src/app/components/SEStatusLabel/SEStatusLabel.tsx

```diff
@@ -15,20 +15,10 @@ export interface SEStatusLabelProps {
   resourceType: "bridge" | "processor";
   /** When the resource was requested (created or updated) */
   requestedAt: Date;
-  /** Flag to decide if the delayed warning check should run only one time.
-   *  If set to "false", or not set, the component will check every 1s if the
-   *  delayed message should be displayed.
-   */
-  singleDelayedCheck?: boolean;
```

Comment:
Removing this feature from the status label component. It was specifically added to make it work within the processor page, where the data was not refreshed. Now it's not necessary anymore. (If you need more info, [here's a discussion](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/pull/118#discussion_r937813702) about it)